odoo/cli/db.py

```diff
@@ -123,15 +123,25 @@ def run(self, cmdargs):
         dump = subs.add_parser(
             "dump", help="Create a dump with filestore.",
             description="Creates a dump file. The dump is always in zip format "
-                        "(with filestore), to get a no-filestore format use "
-                        "pg_dump directly.")
+                        "(with filestore), to get another format, use "
+                        "dump_format argument.")
         dump.set_defaults(func=self.dump)
         dump.add_argument('database', help="database to dump")
         dump.add_argument(
             'dump_path', nargs='?', default='-',
             help="if provided, database is dumped to specified path, otherwise "
                  "or if `-`, dumped to stdout",
         )
+        dump.add_argument(
+            'dump_format', nargs='?', default='zip',
+            help="if provided, database is dumped used the specified format, "
+                "otherwise defaults to `zip`.\n"
+                "Supported formats are `zip`, `dump` (pg_dump format) ",
+        )
+        dump.add_argument(
+            '-nf', '--no-filestore', action='store_const', default=False, const=True,
+            help="if true, zip database is dumped without filestore (default: false)"
+        )
```

Comment:
In most tool `-nf` means `-n -f`, I veto we make a two-chars single dash option.
Also avoid double-negatives, it is fine to expose a `--no-filestore` option on the cli, but internally it should be stored as `filestore` (and not `no_filestore`).
```suggestion
        dump.add_argument(
            '--no-filestore', action='store_const', dest='filestore', default=True, const=False,
            help="if true, zip database is dumped without filestore (default: false)"
        )
```

Reply:
about this argument, the default behaviour should be with filestore (the current behaviour) and if you don't want filestore you use the flag. The function dump has filestore argument so if it is true, the filestore is dumped. In the cli i used "not args.filestore" to pass correctly the bool to the service .
Is it better to use "no_filestore" as argument here in the cli imho instead of filestore .

What do you think?